packages/storybook8/stories/Components/SendBox/SendBox.stories.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { SendBox as SendBoxComponent } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+import React, { useEffect, useRef } from 'react';
+import { controlsToAdd, hiddenControl } from '../../controlsUtils';
+
+import { AttachmentUploadsExample } from './snippets/AttachmentUploads.snippet';
+import { CustomIconExample } from './snippets/CustomIcon.snippet';
+import { CustomStylingExample } from './snippets/CustomStyling.snippet';
+import { MentionsExample } from './snippets/Mentions.snippet';
+import { SendBoxExample } from './snippets/SendBox.snippet';
+import { SendBoxWithSystemMessageExample } from './snippets/SendBoxWithSystemMessage.snippet';
+
+const Default = (args): JSX.Element => {
+  const timeoutRef = useRef<NodeJS.Timeout>();
+  const delayForSendButton = 300;
+
+  useEffect(() => {
+    // Clean up the timeout when the component unmounts
+    return () => {
+      if (timeoutRef.current) clearTimeout(timeoutRef.current);
+    };
+  }, []);
+
+  return (
+    <div style={{ width: '31.25rem' }}>
+      <SendBoxComponent
+        {...args}
+        onSendMessage={async (message, options) => {
+          timeoutRef.current = setTimeout(() => {
+            alert(`Sent message: "${message}" with options: ${JSON.stringify(options)}`);
+          }, delayForSendButton);
+        }}
+        onTyping={() => {
+          console.log(`Sending typing notifications`);
+          return Promise.resolve();
+        }}
+        systemMessage={args.hasWarning ? args.warningMessage : undefined}
+        attachments={
+          args.hasAttachments
+            ? [{ id: 'f2d1fce73c98', name: 'file1.txt', progress: 1, url: 'https://www.bing.com' }]
+            : undefined
+        }
+        onCancelAttachmentUpload={(attachmentId) => {
+          window.alert(`onCancelAttachmentUpload callback is called for attachment id: "${attachmentId}"`);
+        }}
+      />
+    </div>
+  );
+};
+
+Default.storyName = 'SendBox';
+
+export default {
+  title: 'Components/SendBox/Send Box',
```

Comment:
I think in this case we don't need to put Send Box under a folder. 
![image](https://github.com/Azure/communication-ui-library/assets/9044372/9d97f78c-8213-413e-a093-25379f239bef)

As per the new storybook v8 I think it just needs to be `'Components/Send Box'`
And RichTextSendBox will also sit under `'Components/Rich Text Send Box'`

Reply:
But this is what we are doing right now in our current Storybook. Keep `SendBox` and `RichTextSendBox` under same folder. They re related to each other and should be grouped together.